components/dropdown/test/dropdown-menu.vdiff.js

```diff
@@ -135,4 +136,27 @@ describe('dropdown-menu', () => {
 		await oneEvent(dropdown, 'd2l-dropdown-open');
 		await expect(dropdown).to.be.golden();
 	});
+
+	it('radio-button-close', async() => {
+		const elem = await fixture(html`
+			<div style="background-color: #000000; padding: 250px;">
```

Comment:
Super nit - this is running the test in dark mode, but it doesn't really matter either way since it's just testing functionality